src/FakeItEasy/RootModule.cs

```diff
@@ -188,7 +188,7 @@ public bool TryCreateFakeObject(DummyCreationSession session, Type typeOfFake, D
         private class ArgumentConstraintManagerFactory
             : IArgumentConstraintManagerFactory

```

Comment:
> One option would be to rename to include `Negative`

You mean `Negatable`, right? `NegatableArgumentConstraintManagerFactory` is quite a mouthful, and I don't think the fact that it creates a *negatable* argument constraint manager is relevant. I'd rather keep it as is.